docs/website/docs/command-reference/delete.md

```diff
@@ -0,0 +1,38 @@
+---
+title: odo delete
+sidebar_position: 2
+---
+
+### Deleting a component
+
+To delete a _devfile_ component, you can execute `odo delete`.
+
+```shell
+odo delete
+```
+
+If the component is pushed to the cluster, running the above command will delete the component from the cluster, and it's dependant storage, url, secrets, and other resources.
+If it is not pushed, the command would exit with an error stating that it could not find the resources on the cluster.
+
+Use `-f` or `--force` flag to avoid the confirmation questions. 
+
+### Un-deploying Devfile Kubernetes components
+
+To undeploy the Devfile Kubernetes component deployed with `odo deploy` from the cluster, you can execute the `odo delete` command with `--deploy` flag:
```

Comment:
It is possible to push several resources wth odo deploy, and will probably be the normal case.

```suggestion
To undeploy the Devfile Kubernetes components deployed with `odo deploy` from the cluster, you can execute the `odo delete` command with `--deploy` flag:
```